### ISSUE TITLE
Newly deployed n8n start fails on MySQL when table prefix specified 

### ISSUE THREAD

**Issue by artakpetrosyan** [n8n-io/n8n#4011]
When n8n is configured to use MySQL DB, and the table prefix is not empty (for example DB_TABLE_PREFIX=INSTANCE1_) n8n fails to start with SQL error. 

I think the problem is in a missed table prefix for the index name in this function: [https://github.com/n8n-io/n8n/blob/aec5c26564a4cb9b9bb8060276c63487ac4edd67/packages/cli/src/databases/migrations/mysqldb/1644424784709-AddExecutionEntityIndexes.ts#L7-L41](https://github.com/n8n-io/n8n/blob/aec5c26564a4cb9b9bb8060276c63487ac4edd67/packages/cli/src/databases/migrations/mysqldb/1644424784709-AddExecutionEntityIndexes.ts#L7-L41 )

1. Create a new schema in MySQL and use it as a value for DB_MYSQLDB_DATABASE as well as DB type and another connection config.
2. Set not empty value for DB_TABLE_PREFIX
3. start n8n ( used docker image  - ```docker compose up```, but it should fail on just n8n start too)
```
Migration "AddExecutionEntityIndexes1644424784709" failed, error: Can't DROP 'IDX_c4d999a5e90784e8caccf5589d'; check that column/key exists
There was an error initializing DB: "Can't DROP 'IDX_c4d999a5e90784e8caccf5589d'; check that column/key exists"

Stopping n8n...
```

**Expected behavior**
Successfully initialize DB and run.

**Environment (please complete the following information):**

- OS: Ubuntu Linux 16 (not important)
- n8n Version 0.193.2 but it fails on older versions too
- Node.js Version 14
- Database system MySQL
- Operation mode [`own`, `queue`] not important

**Comment by Joffcom** [comment]
Hey @artakpetrosyan,

Thanks for reporting this one, I will take a look this afternoon and see if I can reproduce this one.

**Comment by Joffcom** [comment]
Hey @artakpetrosyan,

Took a quick look and managed to reproduce the issue so I have popped in a PR that should fix this in the future. Thanks again for bringing this one to our attention 👍🏻

**Comment by janober** [comment]
Fix of @Joffcom got merged and will be released with the next version.

**Comment by janober** [comment]
Fix got released with `n8n@0.193.4`

**Comment by artakpetrosyan** [comment]
Thank you!
